jvb/src/main/kotlin/org/jitsi/videobridge/AudioSubscription.kt

```diff
@@ -19,7 +19,7 @@ package org.jitsi.videobridge
 import org.jitsi.videobridge.message.ReceiverAudioSubscriptionMessage
 import org.jitsi.videobridge.relay.AudioSourceDesc
 
-class AudioSubscription() {
+class AudioSubscription(private val conference: Conference) {
```

Comment:
I'm not a fan of tying the basic `AudioSubscription` class to an entire `Conference`. You can introduce a simple `interface` that `Conference` implements. But we can clean that up at a later stage, as the code might need to change for other reasons.